community.md

```diff
@@ -7,11 +7,12 @@ title: Community
 
 ### The xmonad community
 
+*   [the irc channel](http://haskell.org/haskellwiki/IRC_channel): `#xmonad@chat.freenode.org` (join it via [webchat](https://webchat.freenode.net/))
+*   [the matrix channel](https://matrix.to/#/#freenode_#xmonad:matrix.org): this is linked to the above IRC channel
```

Comment:
I wouldn't call this **the** matrix channel, as this is in fact still very much "the irc channel", just through a bridge. There *is* **a** separate matrix room (channel) `#xmonad:matrix.org` which isn't very active and which could possibly later be linked to the IRC bridged one, but it's not right now, but it's the first (and possibly only) one that appears when searching for xmonad. So a more correct wording would be something like "join it via webchat or matrix" or a separate bullet point named "matrix bridge to the irc channel".